translate/storage/fpo.py

```diff
@@ -329,10 +328,6 @@ def hasplural(self):
         source = self.source
         return isinstance(source, multistring) and len(source.strings) > 1
 
-    def parse(self, src):
-        raise DeprecationWarning("Should not be parsing with a unit")
```

Comment:
It is safe to remove `DeprecationWarning` as well since it is only used here and in some other commented line.